lib/client/db/dbcmd/dbcmd_test.go

```diff
@@ -902,13 +867,10 @@ func TestConvertCommandError(t *testing.T) {
 			require.NoError(t, err)
 
 			// make sure the expected test bin is the command bin we got
-			require.Equal(t, cmd.Path, tt.wantBin)
-
-			out, err := c.options.exe.RunCommand(cmd.Path)
```

Comment:
This test shouldn't have depended on `fakeExecer.RunCommand`. `RunCommand` is a method on this interface only to enable tests of mysql-related behavior of `dbcmd`. In certain cases, `RunCommand` is used to check which flavor of mysql is available in the system.